pkg/apis/machine/validation/awsmachineclass.go

```diff
@@ -116,18 +117,49 @@ func validateAWSClassSpecTags(tags map[string]string, fldPath *field.Path) field
 
 func validateBlockDevices(blockDevices []machine.AWSBlockDeviceMappingSpec, fldPath *field.Path) field.ErrorList {
 	allErrs := field.ErrorList{}
-	if len(blockDevices) > 1 {
-		allErrs = append(allErrs, field.Required(fldPath.Child(""), "Can only specify one (root) block device"))
-	} else if len(blockDevices) == 1 {
-		if blockDevices[0].Ebs.VolumeSize <= 0 {
-			allErrs = append(allErrs, field.Required(fldPath.Child("ebs.volumeSize"), "Please mention a valid ebs volume size"))
+
+	if 0 == len(blockDevices) {
```

Comment:
Do we need this new check? Do we know what is the default behaviour in case no block devices are mentioned?

Reply:
I've tested, if you don't provide any block device, AWS will create one for you and mount as root partition. 
I will remove the condition.